serf/config.go

```diff
@@ -242,6 +242,10 @@ type Config struct {
 	// Merge can be optionally provided to intercept a cluster merge
 	// and conditionally abort the merge.
 	Merge MergeDelegate
+
+	// UserEventSizeLimit is maximum byte size limit of user event `name` + `payload` in bytes.
+	// It's optimal to be relatively small, since it's going to be gossiped through the cluster.
```

Comment:
Same as above.

Reply:
Updated in 151d9a86e24c8a380f2c9bce17a00d9abd2a9a69